tskv/src/memcache.rs

```diff
@@ -334,8 +394,11 @@ impl SeriesData {
                 None => continue,
                 Some(index) => *index,
             };
-            for row in item.rows.iter_mut() {
+            let mut rowdata_vec: Vec<RowData> = item.rows.get_ref_rows().iter().cloned().collect();
+            item.rows.clear();
+            for row in rowdata_vec.iter_mut() {
                 row.fields.remove(index);
+                item.rows.insert(row.clone());
             }
```

Comment:
因为 get_ref_rows() 返回的是不可变引用，所以无法使用iter_mut方法。
增加一个 get_mut_rows()就能使用iter_mut()方法。

-------------------
Because get_ref_rows() returns an immutable reference, the iter_mut method cannot be used.
Add a get_mut_rows() to use the iter_mut() method.

Reply:
一个有序的结构，按理来说只能插入，不能原地修改。
原地修改会导致有序的结构可能会变得无序。

Reply:
OrderedSkipList not provided for iter_mut